src/lib/middleware/with-route-spec.ts

```diff
@@ -51,6 +61,10 @@ export const withRouteSpec = createWithRouteSpec({
     pat_with_workspace: withAccessToken({ is_workspace_id_required: true }),
     pat_without_workspace: withAccessToken({ is_workspace_id_required: false }),
     console_session: withSessionAuth({ is_workspace_id_required: true }),
+    user_session: withSessionAuth({ is_workspace_id_required: true }),
+    user_session_without_workspace: withSessionAuth({
+      is_workspace_id_required: false,
+    }),
```

Comment:
What is a user session?

Reply:
I just saw them being used in [seam-connect](https://github.com/seamapi/seam-connect/blob/9b14d85d9c67c9bf8fb8b68a717b34c5b3294bc5/pages/api/public/workspaces/list.ts#L16-L17) and decided to add them. I wasn't aware those are legacy. My bad.

Reply:
Also, this [test](https://github.com/seamapi/javascript-http/pull/150/files#diff-9b4289ec4b49ae2a269c4316344c432aeb0ea3af624ca639427d89c324d65389R106-R122) uses console session auth to request /workspaces/list but the endpoint doesn't support such an auth method on [seam-connect](https://github.com/seamapi/seam-connect/blob/9b14d85d9c67c9bf8fb8b68a717b34c5b3294bc5/pages/api/public/workspaces/list.ts#L14-L19).
Should console session auth be added to /workspaces/list?

Reply:
Removed user session auth 71c02182b65dc374e03a8182d0ed5abd3c1f26f2

Reply:
@razor-x If user_session is legacy, what auth method should handle JWTs in /workspaces/list endpoint to make this [test](https://github.com/seamapi/javascript-http/pull/150/files#diff-9b4289ec4b49ae2a269c4316344c432aeb0ea3af624ca639427d89c324d65389R106-R122) pass?